pyccel/parser/semantic.py

```diff
@@ -1114,9 +1114,8 @@ def _handle_PyccelOperator(self, expr, **settings):
     def _visit_PyccelAdd(self, expr, **settings):
         args = [self._visit(a, **settings) for a in expr.args]
         if isinstance(args[0], (TupleVariable, PythonTuple, Tuple, List)):
-            tuple_args = []
-            for a in args:
-                tuple_args += a.get_vars() if isinstance(a, TupleVariable) else a.args
+            get_vars = lambda a: a.get_vars() if isinstance(a, TupleVariable) else a.args
+            tuple_args = [ai for a in args for ai in f(a)]
```

Comment:
I'm afraid there is a typo: `f` >> `get_vars`.

Reply:
Arrr, stupid mistake ^^